### DISCUSSION TITLE
STaR: Bootstrapping Reaosning With Reasoning and other useful papers.

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/Raspberry#6] [Research]
STAR: https://arxiv.org/abs/2203.14465

CoT papers: https://www.perplexity.ai/search/can-you-find-the-first-chain-o-BEonAvRWSOqyArODXYOkIQ 

Graph of Thought: https://arxiv.org/abs/2305.16582 

More CoT reasoning papers: https://chatgpt.com/share/66e442ff-8ffc-800a-a95b-0ba8e01aa56e

**Comment by andreaskoepf** [comment]
Also can recommend the already quite extensive [open-thought/system-2-research](https://github.com/open-thought/system-2-research) research material link list ;-).

**Reply by russellballestrini** [reply]
Thanks!
